HangFire.Autofac/RegistrationExtensions.cs

```diff
@@ -25,8 +26,31 @@ public static class RegistrationExtensions
             InstancePerBackgroundJob<TLimit, TActivatorData, TStyle>(
             [NotNull] this IRegistrationBuilder<TLimit, TActivatorData, TStyle> registration)
         {
-            if (registration == null) throw new ArgumentNullException("registration");
```

Comment:
There should still be a check for argument here:
`if (registration == null) throw new ArgumentNullException("registration");`